cloud-consumption/src/main/java/com/sequenceiq/consumption/service/ConsumptionService.java

```diff
@@ -121,9 +121,9 @@ public List<Consumption> findAllConsumption() {
     public List<Consumption> findAllStorageConsumptionForEnvCrnAndBucketName(String environmentCrn, String storageLocation) {
         List<Consumption> consumptionsForEnv = consumptionRepository.findAllStorageConsumptionByEnvironmentCrn(environmentCrn);
         try {
-            String bucketName = cloudStorageLocationUtil.getS3BucketName(storageLocation);
+            String bucketName = awsS3ConsumptionCalculator.getObjectId(storageLocation);
```

Comment:
Circling back to my earlier comment in `AwsS3ConsumptionCalculator`, we should find a better general name for this method. Depending on whether the intent is to express the purpose here (e.g. `getStorageAggregationKey`) or the object storage layout (e.g. `getStorageLocationRoot()`), there are many choices. :smile: